test/test_main.cpp

```diff
@@ -26,8 +34,54 @@ void Test_Main::initTest()
  */
 void Test_Main::testOpenDB()
 {
-    Database *db = new Database("test.db", "QSQLITE");
+    db = new Database("test.db", "QSQLITE");
+    QVERIFY(db->open());
+}
+
+void Test_Main::testNumberStadiums()
+{
     QVERIFY(db->open());
+    if(db->isOpen())
+    {
+        QVERIFY(db->getNumberOfStadiums() == 30);
+    }
+
+}
+
+void Test_Main::createGraph()
+{
+    QSqlQuery query;
+    Graph testGraph;
+
+    QVERIFY(db->open());
+    if(db->isOpen()) {
+
+        qDebug() << "Creating the graph";
+        testGraph.createGraph(db);
+    }
+
+    QVERIFY(testGraph.getNumberVertices() == 30);
```

Comment:
This line causes a build error. 

`/home/ozyx/git/subtle-cosmic-moose/test/test_main.cpp:63: error: 'class Graph' has no member named 'getNumberVertices'
     QVERIFY(testGraph.getNumberVertices() == 30);
                       ^`

Gonna try to set up travis soon if I can.